src/components/MediaPlayer.vue

```diff
@@ -143,7 +143,7 @@ export default {
     <div class="media__controls">
       <div class="media__player">
         <button
-          title="playback rate"
+          :title="`${isPlaying ? 'Pause' : 'Play'}`"
```

Comment:
I like the use of title for giving us a free visual indication of what the button does. Let's also add `aria-label` as it is the more accessible attribute for screen readers, while the `title` attribute is less reliable. 

references:
- https://web.dev/button-name/
- https://www.w3.org/WAI/tutorials/forms/labels/#using-the-title-attribute